components/src/dynamo/vllm/omni/stage_worker.py

```
@@ -483,6 +561,40 @@ def _connector_key(from_stage: int, to_stage: int) -> tuple[str, str]:
     return (str(from_stage), str(to_stage))
 
 
+def _uses_nixl_connector(stage_configs_path: str, stage_configs: list[Any]) -> bool:
+    """Check if any stage connector uses NixlConnector."""
+    try:
+        with open(stage_configs_path) as f:
+            deploy_config = yaml.safe_load(f) or {}
+    except (OSError, Exception):
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)